docs/cookbook/presigned-get-download-from-browser.md

```diff
@@ -0,0 +1,51 @@
+# Using pre-signed URLs to download via the browser [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Instead of using an S3-compatible client to download files, you can pre-sign a
+request to download a specified file from the browser, even if its bucket is
+private. The following is an annotated example of how this can be used in a
+Node.js application, using [minio-js](https://github.com/minio/minio-js).
+
+This application will work out of the box, just copy each piece together into a
+file and run it.
+
+## Code
+
+```javascript
+const Minio = require('minio')
+
+var client = new Minio({
+    endPoint: 'play.minio.io',
+    port: 9000,
+    secure: true,
+    accessKey: 'Q3AM3UQ867SPQQA43P2F',
+    secretKey: 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG'
+})
+```
+
+In order to sign the request, we need to log in to the database. This
+creates a new `client` instance, with which you can download and upload files,
+among [much more](https://github.com/minio/minio-js/blob/master/docs/API.md).
+
+These are real credentials to an example Minio server — try it out!
+
+```javascript
+const http = require('http')
+
+http.createServer((req, res) => {
+
+    client.presignedGetObject('pictures', 'house.png', (err, url) => {
+        if (err) throw err
+
+        res.redirect(url)
+    })
+
+}).listen(8080)
+```
+
+[This function](https://docs.minio.io/docs/javascript-client-api-reference#presignedGetObject)
+creates the URL we can use to download `pictures/house.png`. The link will
+automatically expire after 7 days — this can be adjusted using the optional
+`expiry` argument.
+
```

Comment:
you can show a client side JS code too that can run in browser, you can just demo GET of a text file and do console.log - that will do